foodsaving/users/api.py

```diff
@@ -42,6 +42,9 @@ def get_permissions(self):
             self.permission_classes = (AllowAny,)
         elif self.action in ('list', 'retrieve'):
             self.permission_classes = (IsAuthenticated,)
+        else:
+            # Issue 232 default permission classes should always be returned
+            self.permission_classes += self.__class__.permission_classes
```

Comment:
I don't understand what this line doing. Sorry for not being helpful, I'm still travelling and can just occasionally check my phone.
However, I think this code doesn't need to be changed, or what do you want to achieve with it?